codespell_lib/data/dictionary.txt

```diff
@@ -23963,6 +23966,7 @@ self-comparisson->self-comparison
 self-contianed->self-contained
 self-referencial->self-referential
 self-refering->self-referring
+selt->set, self, sold,
```

Comment:
Suggest seat.

I'm struggling to see sold though?

From looking at grep.app I think it should be self and then set.

Reply:
`He selt it last week` a foreign speaker could make this mistake.

Reply:
Okay, I thought that would be more likely to be "selled it" (do you want to add that too if it's not there.

Can you swap self and set though please.